flowey/flowey_lib_hvlite/src/_jobs/check_openvmm_hcl_size.rs

```diff
@@ -0,0 +1,163 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+//! Compares the size of the OpenHCL binary in the current PR with the size of the binary from the last successful merge to main.
+
+use crate::artifact_openhcl_igvm_from_recipe_extras;
+use crate::artifact_openhcl_igvm_from_recipe_extras::OpenhclIgvmExtras;
+use crate::build_openhcl_igvm_from_recipe;
+use crate::build_openhcl_igvm_from_recipe::OpenhclIgvmRecipe::X64;
+use crate::build_openvmm_hcl::OpenvmmHclBuildProfile::OpenvmmHclShip;
+use crate::run_cargo_build::common::CommonArch;
+use crate::run_cargo_build::common::CommonTriple;
+use flowey::node::prelude::*;
+use flowey_lib_common::download_gh_artifact;
+use flowey_lib_common::gh_merge_commit;
+use flowey_lib_common::gh_workflow_id;
+
+flowey_request! {
+    pub struct Request {
+        pub target: CommonTriple,
+        pub done: WriteVar<SideEffect>,
+        pub artifact_dir_openhcl_igvm_extras: ReadVar<PathBuf>,
+        pub pipeline_name: String,
+    }
+}
+
+new_simple_flow_node!(struct Node);
+
+impl SimpleFlowNode for Node {
+    type Request = Request;
+
+    fn imports(ctx: &mut ImportCtx<'_>) {
+        ctx.import::<crate::build_xtask::Node>();
+        ctx.import::<crate::git_checkout_openvmm_repo::Node>();
+        ctx.import::<download_gh_artifact::Node>();
+        ctx.import::<gh_merge_commit::Node>();
+        ctx.import::<gh_workflow_id::Node>();
+        ctx.import::<build_openhcl_igvm_from_recipe::Node>();
+        ctx.import::<artifact_openhcl_igvm_from_recipe_extras::publish::Node>();
+    }
+
+    fn process_request(request: Self::Request, ctx: &mut NodeCtx<'_>) -> anyhow::Result<()> {
+        let Request {
+            target,
+            done,
+            pipeline_name,
+            artifact_dir_openhcl_igvm_extras,
+        } = request;
+
+        let xtask = ctx.reqv(|v| crate::build_xtask::Request {
+            target: target.clone(),
+            xtask: v,
+        });
+        let openvmm_repo_path = ctx.reqv(crate::git_checkout_openvmm_repo::req::GetRepoDir);
+
+        let (read_built_openvmm_hcl, built_openvmm_hcl) = ctx.new_var();
+        let (read_built_openhcl_boot, built_openhcl_boot) = ctx.new_var();
+        let (read_built_openhcl_igvm, built_openhcl_igvm) = ctx.new_var();
+
+        let gh_token = ctx.get_gh_context_var().global().token();
+
+        ctx.req(build_openhcl_igvm_from_recipe::Request {
```

Comment:
you're going to want to hit `build_openvmm_hcl` directly, and sidestep building all this other stuff we don't currently care about.

note that, in doing so, you won't be able to pass a recipe directly, and have it "just work". instead - you'll need to call `recipe_details` on a baseline recipe we want to verify the size of, and manually plumb through the returned openhcl_vmm-specific config data to the `build_openvmm_hcl` node